kodilla-exception/src/main/java/com/kodilla/exception/test/FirstChallenge.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.exception.test;
+
+public class FirstChallenge {
+    public double divide(double a, double b) throws ArithmeticException {
+        if(b == 0){
+            throw new ArithmeticException();
+        }
+        return a / b;
+    }
+
+    public static void main(String[] args) {
+
+        FirstChallenge firstChallenge = new FirstChallenge();
+        try {
+            double result = firstChallenge.divide(3, 0);
+            System.out.println(result);
+        } catch (ArithmeticException e) {
+            System.out.println("Błąd: Dzielenie przez 0");
+        } finally {
+            System.out.println("Pamiętaj cholero nie dziel przez zero");
```

Comment:
Finally sie wykona zawsze. Wiec jakktos poda dobre dane, to po co mu wypisywac ten tekst?